### DISCUSSION TITLE
Migrating from Obsidian - Need some guidance

### DISCUSSION THREAD

**Discussion by itsallyours** [TriliumNext/Trilium#3507] [Q&A]
I have been using Obsidian for years and trying to adopt Trilium.  There are many great features of Trilium.
After working thru import topics, I am looking to understand best practices.

1. Migration from folderless structure
2. Deferred creation of notes

### Folderless Structure
When I used Obsidian, I effectively had only 2 folders. One for daily notes and one for everything else.  I got a firm grasp on daily notes in Trilium but not sure about all of the other notes.  I basically had a folderless structure...with a few thousand notes in that "everything else" folder.  I used a few MOC (Maps of Content:  see https://www.youtube.com/watch?v=WUq8Pun28FI) to create a few launch points but for the most part, notes just organically meshed up.  

In Trilium, I first started by just placing all "people" notes under a People note.  In my situation, it was still 500+ notes.  There's not a simple layout of Coworkers, Vendors, Family, Friends, etc.  I was even thinking about grouping by 1st letter of their last name to keep the node length manageable.  That's just People,  I have the same problem with other nouns Schools, Companies, Cities, Products, etc.  With Obsidian, I just didn't have to structure it.   They all get intermixed with How To articles and topics.  

How do you handle shallow, wide trees?  Can I hide sub nodes on certain nodes?  Understanding that I would never browse to it...CTRL-J or link would get me there.

Would something like this work better?
https://axle.design/systemics-and-design-principles-in-support-of-tiago-forte-s-para-framework

I have read up on and currently use Cloning. 

### Deferred creation of notes
With Obsidian, you can defer creation of notes and it would just link up later when you do create them.  (Wikipedia and many other platforms do this)

Imagine taking meeting notes on your Daily Note.  I like to list the people who are in attendance.  Often, there are new people.  What is the Trilium approach?  Right now, I do the following:

1. select the person
2. CTRL-L
3. Select the existing note or create a new note
4. If create, then I clone the empty person note to my People folder 
6. Resume taking notes in the original note
7. Repeat for each Person, product, etc.

I don't mind having empty notes out there that I can improve later that are already pre-linked.

Is there a better approach?  Maybe some type of macro/code approach to simplify?

**Comment by zadam** [comment]
> Can I hide sub nodes on certain nodes? Understanding that I would never browse to it...CTRL-J or link would get me there.

You could use `#archived` label, potentially set as inheritable for such notes.

> With Obsidian, you can defer creation of notes and it would just link up later when you do create them. (Wikipedia and many other platforms do this)

This is not possible in Trilium. 

> Is there a better approach? 

Similar to your process, maybe I would use in place linking - type @, type the note name, "create and link child" ... I would clone the notes in batch (selecting all) later when I have time.
